routes/channel.js

```diff
@@ -64,13 +64,13 @@ function getEventAggregates(req, res, next) {
 async function getList(req, res, next) {
 	const { CHANNELS_FIND_LIMIT } = cfg
 	// assign 0 default value
-	const { page = 0 } = req.query
-
+	const { page = 0, creator, validUntil } = req.query
 	const channelsCol = db.getMongo().collection('channels')
 	const skip = page && parseInt(page, 10) * CHANNELS_FIND_LIMIT
 	let query = {
-		validUntil: { $gt: Math.floor(Date.now() / 1000) }
+		validUntil: { $gt: parseInt(validUntil, 10) || Math.floor(Date.now() / 1000) }
 	}
+	query = (creator && { ...query, creator }) || query
```

Comment:
this syntax is unintuitive, you can just `query = creator ? ... : ...` which is more clear